### ISSUE TITLE
Tests not runnable on case-insensitive systems

### ISSUE THREAD

**Issue by viraptor** [KenKundert/emborg#69]
Since the tests do things like:
```
            rm("configs .config .local repositories configs.symlink".split())
>           cp("CONFIGS", "configs")
```
they cannot be run on case-insensitive filesystems like macos. This could be fixed by using a name like `configs.fixture` rather than relying on `CONFIGS` being different from `configs`.

This came up when automatically testing in nixpkgs: https://hydra.nixos.org/build/212351260/nixlog/3

**Comment by KenKundert** [comment]
Thanks for reporting this.  I'll get back to you when I've resolved the issue.

**Comment by KenKundert** [comment]
I believe this to be resolved in the latest release (1.35), which is now on pypi.

**Comment by viraptor** [comment]
Thanks. That solved the config naming issues. There's till other stuff that fails now, but I'll try to debug that separately.
